python/tk_multi_launchapp/base_launcher.py

```diff
@@ -294,6 +294,7 @@ def _launch_app(
             if self._tk_app.engine.has_ui:
                 from ..not_found_dialog import show_generic_error_dialog
 
+                self._tk_app.log_error(msg)
```

Comment:
If we want to print this log whether or not there is UI, you can move the existing log line before the if block to avoid code duplication.